.gitmodules

```diff
@@ -5,8 +5,8 @@
 [submodule "ccpp/framework"]
 	path = ccpp/framework
 	url = https://github.com/NCAR/ccpp-framework
-	branch = main
+	branch = master
```

Comment:
this must be `main`

Reply:
Jongil, would you please fix this?

Reply:
I don't understand what's the problem and please direct me to fix it.

Reply:
In the file .gitmodules in your fv3 branch han_p7, you are using "master" branch for ccpp-framework, please change it back to "main".

Reply:
In the file .gitmodules in my fv3 branch han_p7, I changed "master" to "main" and commit and pushed back. Should I also update my ufs-weather-model branch (ufs_hanp7)? Also, do I need to re-do the RT run with this ccpp-framework?

Reply:
No, because the hash of ccpp-framework that fv3atm uses is still the same.